app.py

```diff
@@ -88,7 +89,17 @@ def route(album, artist):
 
 @app.route("/artists/<artist_name>", methods=("GET", "POST"))
```

Comment:
Oh, I also wanted to say: 

I _really_, _really_, _really_ like this pattern here. 

Currently in the search feature that I worked on/have been working on the template rendering takes place in the home() function route. This means that when a search takes place the path doesn't appear in the url even though a new template has been loaded. This makes it somewhat difficult to test. I like your routing much better and will probably adapt the search feature around what you have implemented here. It simply makes more sense.